tests/integration/testutils/test_utils.go

```diff
@@ -0,0 +1,320 @@
+//go:build integration
+// +build integration
+
+/*
+ * Copyright 2021 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package testutils
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/newrelic/nri-cassandra/tests/integration/jsonschema"
+	"github.com/stretchr/testify/assert"
+	"github.com/testcontainers/testcontainers-go"
+)
+
+// GetTestName returns the name of the running test.
+func GetTestName(t *testing.T) interface{} {
```

Comment:
is it useful to improve readability of `t.Name()`?

Reply:
leftover from previous helpers file, not really needed, removing it